src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
@timbussmann What do you mean with backporting? We are talking about an issue on 5.2.14, that was categorized as a bug. The current fix is a patch according to semver. Its not fully backwards compatible. Fixing this behavior but also making it behave by default to be backwards compatible but can be configured means an API changes thus requires not 5.2.15 but 5.3.0.